reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -630,36 +631,43 @@ class StaticHtmlReporter : Reporter {
         val renderer = HtmlRenderer.builder().build()
         unsafe { +renderer.render(document) }
     }
-}
 
-private fun DIV.licenseLink(license: String) {
-    val licenseResourcePath = getLicenseResourcePath(license)
-    val sha1Git = HashAlgorithm.SHA1_GIT.calculate(licenseResourcePath) ?: return +license
+    private fun DIV.licenseLink(license: String) {
+        val licenseResourcePath = getLicenseResourcePath(license)
+        val sha1Git = licensesSha1.getOrPut(
+            license,
+            { HashAlgorithm.SHA1_GIT.calculate(licenseResourcePath) ?: license }
```

Comment:
Please move the lamda argument out of the parameter list, like

    val sha1Git = licensesSha1.getOrPut(license) {
        HashAlgorithm.SHA1_GIT.calculate(licenseResourcePath) ?: license
    }